core/lib/config/src/configs/en_config.rs

```diff
@@ -0,0 +1,40 @@
+use std::num::{NonZeroU32, NonZeroU64, NonZeroUsize};
+
+use serde::Deserialize;
+use zksync_basic_types::{
+    commitment::L1BatchCommitmentMode, url::SensitiveUrl, L1ChainId, L2ChainId,
+};
+
+#[derive(Debug, Clone, PartialEq, Deserialize)]
+pub struct Pruning {
+    pub enabled: bool,
+    pub chunk_size: Option<u32>,
+    pub removal_delay_sec: Option<NonZeroU64>,
+    pub data_retention_sec: Option<u64>,
+}
+
+#[derive(Debug, Clone, PartialEq, Deserialize)]
+pub struct SnapshotRecovery {
+    pub enabled: bool,
+    pub postgres_max_concurrency: Option<NonZeroUsize>,
+    pub tree_chunk_size: Option<u64>,
+}
+
+/// Temporary config for initializing external node, will be completely replaced by consensus config later
+#[derive(Debug, Clone, PartialEq, Deserialize)]
+pub struct ENConfig {
+    // Genesis
+    pub l2_chain_id: L2ChainId,
+    pub l1_chain_id: L1ChainId,
+    pub l1_batch_commit_data_generator_mode: L1BatchCommitmentMode,
```

Comment:
Just to understand better: There's intersection in these 3 params with `GenesisConfig`. Is there nothing that can be done to share these 3 params?

Reply:
So, yes they are part of the genesis config, I've mentioned it here, just because with consensus we will remove this file and we don't need to have full genesis